jobs/ClusterHQ_Senior_Lead_QA

```diff
@@ -0,0 +1,74 @@
+title: Senior / Lead Software Test Engineer
+company: ClusterHQ
+url: http://techfolk.co.uk/current-jobs/senior-lead-software-test-engineer-central-bristol-tl103
+location: Bristol, England
+contract: permanent
+contact:
+    name: Andrew Gifford
+    email: hello@techfolk.co.uk
+    phone: +44 (0)117 318 2447
+    ...: ...
+created: 2015-08-04
+tags:
+  - Bristol
+  - England
+  - uk
+  - relocation
+  - QA
+  - testing
+  - software testing
+  - containers 
+  - containerisation
+  - containerization
+  - virtualisation
+  - virtualization
```

Comment:
Thanks for the pointers Daniel. Appreciated. Forgive me stumbling around
like a newborn foal whilst I familiarise myself! J



*From:* Daniel Pope [mailto:notifications@github.com]
*Sent:* 04 August 2015 14:35
*To:* pythonjobs/jobs <jobs@noreply.github.com>
*Cc:* Andrew Gifford <hello@techfolk.co.uk>
*Subject:* Re: [jobs] Create ClusterHQ_Senior_Lead_QA (#103)



In jobs/ClusterHQ_Senior_Lead_QA
<https://github.com/pythonjobs/jobs/pull/103#discussion_r36187530>:

> +    phone: +44 (0)117 318 2447

> +    ...: ...

> +created: 2015-08-04

> +tags:

> +  - Bristol

> +  - England

> +  - uk

> +  - relocation

> +  - QA

> +  - testing

> +  - software testing

> +  - containers

> +  - containerisation

> +  - containerization

> +  - virtualisation

> +  - virtualization

I don't think we should duplicate all the tags just to handle US English.
I'd be happy to go with British English spellings.

—
Reply to this email directly or view it on GitHub
<https://github.com/pythonjobs/jobs/pull/103/files#r36187530>.